jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/PluginConfigurationProcessor.java

```diff
@@ -506,7 +506,8 @@ static JavaContainerBuilder getJavaContainerBuilderWithBaseImage(
     }
 
     // Verify Java version is compatible
-    String prefixRemoved = baseImageConfig.replaceFirst(".*://", "");
+    String[] splits = baseImageConfig.split("://");
+    String prefixRemoved = splits[splits.length - 1];
```

Comment:
In general, Guava's Splitter gives more expected behavior.
https://github.com/google/guava/wiki/StringsExplained#splitter

Reply:
Thanks for the link!  Updated to Guava Splitter.